core/src/main/scala/org/apache/spark/BarrierCoordinator.scala

```diff
@@ -108,7 +108,7 @@ private[spark] class BarrierCoordinator(
 
     // The request method which is called inside this barrier sync. All tasks should make sure
```

Comment:
@Ngone51 Perhaps change the comment here to reflect the change ?

Reply:
Sure, thanks.